lib/sort_by_length.rb

```diff
@@ -1,7 +1,20 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(n)
+# I think this is an insertion sort (or that's what I wanted it to be haha) - is that true?
 def sort_by_length(my_sentence)
```

Comment:
This is closer to a bubble sort, as it's making swaps for the word to insert each iteration of the inner loop.  Look at the textbook curriculum for examples of the sorts and look at how yours compares.